ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/structure/header/header.html

```diff
@@ -0,0 +1,59 @@
+<!-- Header Component Start -->
+<sly
+  data-sly-use.navHeader="com.aembootcamp.core.models.HeaderModel"
+  data-sly-test.empty="${!navHeader.fileReference}"
+  data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+  data-sly-call="${template.placeholder @ isEmpty=empty}"
+/>
+<sly data-sly-test="${!empty}">
+  <div class="container responsivegrid">
+    <div class="container responsivegrid cmp-layoutcontainer--header">
+      <div class="cmp-container">
+        <div class="aem-Grid cmp-container--layout">
+          <div
+            class="image cmp-image--logo aem-GridColumn"
+            style="display: inline-block"
+          >
+            <div class="cmp-image">
+              <a
+                class="cmp-image__link"
+                href="${navHeader.logoLink @extension='html'}"
+              >
+                <img
+                  src="${navHeader.fileReference}"
+                  loading="lazy"
+                  class="cmp-image__image"
+                  itemprop="contentUrl"
+                  alt="${navHeader.altText}"
+                />
+              </a>
+            </div>
+          </div>
+          <div
+            class="navigation cmp-navigation--header"
+            style="display: inline"
+          >
+            <nav class="cmp-navigation">
+              <ul
+                data-sly-repeat="${navHeader.headerNavigationItemsList}"
```

Comment:
repeat should be on li tag instead of ul
please review documentation 
https://github.com/adobe/htl-spec/blob/master/SPECIFICATION.md#227-repeat